src/js/components/Header.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+class Header extends React.Component {
+
+  getIcon(icon, className) {
+    if (icon == null) {
+      return null;
+    }
+
+    return (
+      <div className="media-object-item">
+        <div className={className}>
+          {icon}
+        </div>
+      </div>
+    );
+  }
+
+  getTitle(title, className) {
+    if (title == null) {
+      return null;
+    }
+
+    return (
+      <h1 className={className}>
+        {title}
+      </h1>
+    );
+  }
+
+  getSubTitle(subTitle) {
+    if (subTitle == null) {
+      return null;
+    }
+
+    return (<div>{subTitle}</div>);
+  }
+
+  render() {
+    let props = this.props;
```

Comment:
`let {props} = this;` ?

It's nice to write it always in this style, so you could easily add other variables like `state` at a later point, if you need it.